syz-fuzzer/proc.go

```diff
@@ -117,15 +117,10 @@ func (proc *Proc) executeProgram(req *flatrpc.ExecRequest, wait time.Duration) (
 		var output []byte
 		var info *flatrpc.ProgInfo
 		var hanged bool
-		// On a heavily loaded VM, syz-executor may take significant time to start.
-		// Let's do it outside of the gate ticket.
 		err := proc.env.RestartIfNeeded(req.ExecOpts)
```

Comment:
Nit: we probably don't need a public RestartIfNeeded method anymore.

Reply:
Hopefully we won't need the ipc package at all soon. So I don't want to do lots of cleanup/refactoring there, if it's not needed for other things.
I wanted to remove ipc gate first b/c it looks like it's one of major performance limiting factors. W/o removing it, it's hard to measure impact of other improvements.